mbed_lstools/lstools_base.py

```diff
@@ -210,8 +210,7 @@ def _update_device_from_htm(self, device):
                             htm_target_id, device['target_id_usb_id'])
             device['target_id'] = htm_target_id
         else:
-            logging.warning("Could not read htm on from usb id %s. "
-                            "Falling back to usb id",
+            logging.warning("No HTM target ID found %s. Falling back to usb id",
```

Comment:
I thought we weren't using `logging` anymore, just the `logger` instance? Looks like `logging` is used a few times in this file.

Reply:
Hmmmmmmmm. Don't know how that dropped out.